internal/provider/namespace_resource.go

```diff
@@ -262,6 +263,14 @@ func (r *namespaceResource) Schema(ctx context.Context, _ resource.SchemaRequest
 				},
 				Computed: true,
 			},
+			"connectivity_rule_ids": schema.ListAttribute{
+				Description: "The IDs of the connectivity rules for this namespace.",
+				Optional:    true,
```

Comment:
Can you add the element type?
```suggestion
				Optional:    true,
				ElementType: types.StringType,
```